rust_src/src/threads.rs

```diff
@@ -24,6 +25,11 @@ impl ThreadStateRef {
     pub fn name(self) -> LispObject {
         LispObject::from(self.name)
     }
+
+    #[inline]
+    pub fn is_alive(self) -> bool {
+        self.m_specpdl != ptr::null_mut()
```

Comment:
Would it be worth instantiating our own copy of `ptr::null_mut()` in one of the Remacs libs and using it to check nulls or is the use of `std::ptr` expected where null checks need to be performed?

Reply:
`ptr::null` and `null_mut` are the idiomatic wau of dealing with pointers instead of checking agsinst zero. It is analagous to `NULL` in C.